core/arch/opnd.h

```diff
@@ -2472,6 +2472,19 @@ DR_API
 bool
 reg_is_simd(reg_id_t reg);
 
+DR_API
+/**
+ * Assumes that \p reg is a DR_REG_ constant.
+ * Returns true iff it refers to an SSE or AVX* register.
+ * In particular, the register must be either an XMM, YMM or
```

Comment:
nit: I think all the other descriptions use lower case xmm, ymm, or zmm.
2nd nit: ", or zmm" I am usually getting caught missing the comma myself :)

Reply:
Done